client/services/tables/tables.service.js

```diff
@@ -13,12 +14,16 @@ angular.module('probedock.tables').factory('tables', function(api) {
         table.pagination.start = table.pagination.start || 0;
         table.pagination.number = table.pagination.number || options.pageSize || 15;
 
-        var params = _.extend({}, options.params, {
+        var params = _.extend({}, options.params, list.params, {
           page: table.pagination.start / table.pagination.number + 1,
           pageSize: table.pagination.number
         });
 
-        $scope.$broadcast(name + '.refresh');
+        if (table.search.filter) {
+          $scope.$broadcast(name + '.filter');
+        } else {
+          $scope.$broadcast(name + '.refresh');
+        }
```

Comment:
Revert this event change as it is no longer used (only keep the contents of the `else`). Also in `updateRecords`.